doc/pages/Manual.md

```diff
@@ -698,3 +866,248 @@ recommended to check the validity and quality of your `opam` files.
 
     This is typically useful to pass appropriate flags to `./configure` scripts,
     depending on what is installed.
+
+- <a id="opamfield-synopsis">`synopsis: <string>`</a>:
+  defines a short description (one line) for the package. This can also be
+  defined as the first line of an external [`descr`](#descr) file.
+
+- <a id="opamfield-description">`description: <string>`</a>:
+  defines a long description (one or more paragraphs) for the package. This can
+  also be defined as the body of an external [`descr`](#descr) file.
+
+- <a id="opamsection-url">`url "{" <url-file> "}"`</a>:
+  defines the URL where the package source can be obtained. This section has
+  contents in the same format as the [`url`](#url) file, and has the same effect
+  as using a separate `url` file.
+
+- <a id="opamfield-setenv">`setenv: [ <environment-update> ... ]`</a>:
+  defines environment variables updates that will be applied upon installing the
```

Comment:
What is the environment variable definition policy whenever multiple package define or extend the same variable name ? Something deterministic that doesn't depend on the install sequence has to be defined otherwise this could become a source of non-reproducible builds (and other user support nightmares).